web3_proxy/tests/test_proxy.rs

```diff
@@ -13,15 +13,15 @@ use web3_proxy::rpcs::blockchain::ArcBlock;
 #[cfg_attr(not(feature = "tests-needing-docker"), ignore)]
 #[test_log::test(tokio::test)]
 async fn it_migrates_the_db() {
-    let x = TestApp::spawn(true).await;
+    let x = TestApp::spawn(31337, true).await;
```

Comment:
was it changed to 999_001_999 now? or is this number correct?

Reply:
31337 is anvil's default network with ETH's CU price. 999_001_999 is just for having a very high CU cost to make tests simpler